sbi/neural_nets/net_builders/mnle.py

```diff
@@ -102,6 +105,10 @@ def build_mnle(
             as z_score_x.
         flow_model: type of flow model to use for the continuous part of the
             data.
+        categorical_model: type of categorical net to use for the discrete part of
+            the data. Can be "made" or "mlp".
```

Comment:
add a note here that MLP works only for 1D / one column.

Reply:
good catch! forgot to remove this, since the MLP was removed anyways!